site/content/user-guide/http-adapter.md

```diff
@@ -183,7 +183,7 @@ Publish some JSON data for device `4711`, indicating that the device will wait f
         * The tenant that the gateway belongs to is not allowed to use this protocol adapter.
         * The device belongs to another tenant than the gateway.
         * The gateway is not authorized to act *on behalf of* the device.
-        * The gateway is disabled.
+  * 404 (Not Found): The request cannot be processed because the device is disabled or does not exist.
```

Comment:
have you actually tried the case of a disabled gateway? According to the Device Registration API this should indeed result in a 403. However, adding the 404 and its description is still correct...

Reply:
No, I didn't check this with gateways.

Reply:
Can you then please update the PR to keep the case of a disabled gateway in the list for the 403 status code? And please  don't forget the other two places where this applies as well ...

Reply:
For gateway mode:
Gateway is disabled: 403
Device is disabled: 404

Changed PR accordingly.